spec.emu

```diff
@@ -23,6 +23,7 @@ copyright: false
     1. If _longest_ is *true*, then
       1. Set _mode_ to ~longest~.
       1. Set _paddingOption_ to ? Get(_options_, *"padding"*).
+      1. If _paddingOption_ is not *undefined* and _paddingOption_ is not an Object, throw a *TypeError* exception.
```

Comment:
if longest isn't true, shouldn't this still validate paddingOption?

Reply:
In general (and in Temporal) I think we've been optimistically reading and validating every option whether they're used or not, to minimize depending on observable lookups later.

Reply:
If there's an option which is only relevant to one mode, it seems wasteful to read that option in other modes.

Reply:
> to minimize depending on observable lookups later

there won't be a later lookup because it's guaranteed to not be needed